addons/l10n_ua/migrations/1.3/pre-update_views_unlink.py

```diff
@@ -0,0 +1,18 @@
+from odoo import api, SUPERUSER_ID
+
+
+def migrate(cr, version):
+    env = api.Environment(cr, SUPERUSER_ID, dict())
+
+    w = env.ref('l10n_ua.view_ua_company_form', raise_if_not_found=False)
+    if w:
+        w.unlink()
+    w = env.ref('l10n_ua.view_partner_form_l1', raise_if_not_found=False)
+    if w:
+        w.unlink()
+    w = env.ref('l10n_ua.view_res_partner_filter_l', raise_if_not_found=False)
+    if w:
+        w.unlink()
+    w = env.ref('l10n_ua.view_users_inherit_form', raise_if_not_found=False)
+    if w:
+        w.unlink()
```

Comment:
Maybe the migration script is in handy for you, but makes no sense for us in the stable version.

Reply:
This is for those who are using our addons now(9.0-12.0) and will migrate to 13.0 later. I can remove it.